src/components/Toolbar.js

```diff
@@ -163,10 +163,6 @@ export default class Toolbar extends Component {
       this.setState({
         show: true
       });
-    } else {
-      this.setState({
-        show: false
-      });
```

Comment:
Ditto comment.  If we're going to be submitting this PR into megadraft, we'll need more info (probably in the commit message) why this existed in the first place and why we don't need it anymore.

Reply:
Ok I added further explanations in the commit message

Reply:
Alright, After playing around with this, the state `show` variable is not needed in componentWillReceiveProps, but it is good that the `shouldUpdatePos` is being set in there, because `componentWillReceiveProps` is only called when props change. If we move this logic to `componentDidUpdate` then it is called on every update of props and state leading to a infinite call. So we need to maybe just move the `handleSetToolbar` to `componentWillReceiveProps`

Reply:
ok the set state has been removed completely from the `componentWillReceiveProps` we are leaving everything else in there